styles/styles.css

```diff
@@ -0,0 +1,278 @@
+* {box-sizing: border-box;}
+
+button, input, select, textarea {
+  font-family: inherit;
+  font-size: 100%;
+}
+.skip-hidden {
+  position: absolute;
+  top: -10000px;
+  left: -10000px;
+  height: 1px;
+  width: 1px;
+  text-align: left;
+  overflow: hidden;
+}
+a.skip-hidden:focus {
+  left: 0;
+  top: 0;
+  width: auto;
+  height: auto;
+  overflow: visible;
+}
+header {
+  display: flex;
+  flex-direction: row;
+  background-image: url(header-image.jpeg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: right;
+  background-color: #0F0C0C;
+  border: solid .1rem;
+  width: 85%;
```

Comment:
this should be 100%, but if you want to keep it from going too wide, you can add a max-width to keep it from going too wide